src/Symfony/Component/DependencyInjection/Loader/schema/dic/services/services-1.0.xsd

```diff
@@ -153,8 +153,8 @@
   </xsd:complexType>
 
   <xsd:complexType name="argument" mixed="true">
-    <xsd:choice maxOccurs="unbounded">
```

Comment:
This allowed having several anonymous services in the same argument while only the first was considered by the loader.